github-for-jira.sd.yml

```diff
@@ -44,6 +44,16 @@ resources:
         - PII/IndirectConfidential # name of GitHub org
         - UGC/PrimaryIdentifier    # references to GitHub entities (commits, pull requests, etc.) and Jira issues
 
+  - type: sqs
+    name: webhooksqs
+    attributes:
+      MaxReceiveCount: 3 # If changing this attribute value, then also change the constant value in MessageRouter
```

Comment:
/nit I have to remove this copy-pasted comment

Reply:
Do we need to address the TODO on [line 149](https://github.com/atlassian/github-for-jira/blob/cc023a76644fbf715b686a2278c46abad1ce60e9/github-for-jira.sd.yml#L149)?